Library/Formula/openvpn.rb

```diff
@@ -42,11 +45,17 @@ def install
     rm "#{share}/doc/openvpn/README.polarssl"
   end
 
-  def caveats; <<-EOS.undent
-    If you have installed the Tuntap dependency as a source package you will
-    need to follow the instructions found in `brew info tuntap`. If you have
-    installed the binary Tuntap package, no further action is necessary.
+  def caveats
+    if MacOS.version < :yosemite
+      puts <<-EOS.undent
```

Comment:
You can use `s` instead of `puts`. See [Node](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/node.rb#L116-L136) for an example.